### PR TITLE
Kl 2021 04 new strings

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
New po files as pushed to transifex and few translations to German pulled from transifex

### CODE REVIEW

File: locale-source/locale/en/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -4580,3 +4581,54 @@ msgstr "Login with social media account"
 #: adhocracy-plus/templates/socialaccount/snippets/provider_list.html:23
 msgid "Login with "
 msgstr "Login with "
+
```
Comment (Rineee) [inline locale-source/locale/en/LC_MESSAGES/django.po:4584]: Did you leave these in on purpose, so translators can copy from here to the new strings with adhocracy+ replaced? Im asking bc unfortunately transifex also shows commented out strings, so normally we should remove them, so translators dont have extra work with it...

Comment (fuzzylogic2000) [reply locale-source/locale/en/LC_MESSAGES/django.po:4584]: Yes, exactly! And I already copied from them for German. We have the other thing in the docs, so will also remember to delete them next time. :)

Comment (Rineee) [reply locale-source/locale/en/LC_MESSAGES/django.po:4584]: Ha, perfect!

**Comment by fuzzylogic2000** [comment]
Ah, I don't get why travis is failing. Problem for Monday...

**Comment by fuzzylogic2000** [comment]
Whoop whoop! Fixed itself!

**Pull request merged**